llvm-bazel/llvm-project-overlay/llvm/tblgen.bzl

```diff
@@ -28,11 +28,7 @@ def gentbl(name, tblgen, td_file, td_srcs, tbl_outs, library = True, **kwargs):
     """
     if td_file not in td_srcs:
         td_srcs += [td_file]
-    includes = []
     for (opts, out) in tbl_outs:
-        outdir = out[:out.rindex("/")]
-        if outdir not in includes:
-            includes.append(outdir)
         rule_suffix = "_".join(opts.replace("-", "_").replace("=", "_").split(" "))
         native.genrule(
             name = "%s_%s_genrule" % (name, rule_suffix),
```

Comment:
Of course GitHub won't let me comment on the actual line, but can you use your macros for the include arguments in `cmd` below?

Reply:
Not really... The command line flag syntax below I believe is subtly different from the C++ syntax. At the least you can't use the joined flag syntax (`-Ifoo`), and unsure how many other differences there might be. I'd much rather the macros I'm adding deal exclusively in compiler flags.

I think I can clean this up a bit though, i'll send a follow-up patch.